test/mocks/packages/package-empty-libzos.test.json

```diff
@@ -0,0 +1,11 @@
+{
+  "contracts": {},
```

Comment:
This rename and others in this folder went wrong: there is a missing `.` before `zos`

Reply:
Or are these files generated by tests, and added by accident? If so, then we should:
- update the `clean` step in the tests to include them
- check out why the . is missing if they were autogenerated